src/NRediSearch/InfoResult.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System.Collections.Generic;
+using StackExchange.Redis;
+
+namespace NRediSearch
+{
+    public class InfoResult
+    {
+        public string IndexName { get; }
+
+        public Dictionary<string, RedisResult[]> Fields { get; } = new Dictionary<string, RedisResult[]>();
+
+        public long NumDocs { get; }
+
+        public long NumTerms { get; }
+
+        public long NumRecords { get; }
+
+        public double InvertedSzMb { get; }
```

Comment:
nomenclature; is the megabit (Mb)? megabyte (MB)? or mebibyte? (MiB)? I suspect the last, but I'm pretty sure it isn't the first, so `Mb` is misleading

Reply:
You suspect correct!

[source](https://github.com/RediSearch/RediSearch/blob/fdbb3b21bb875cdcd798248b6f1346242048cb4c/src/module.c#L188)

Looks like the number of bytes divided by `0x100000`